File: als-server/jvm/src/main/scala/org/mulesoft/als/server/lsp4j/extension/SerializationClientCapabilities.java
package org.mulesoft.als.server.lsp4j.extension;

public class SerializationClientCapabilities {
    private Boolean acceptsNotification;

    public SerializationClientCapabilities(Boolean acceptsNotification){
        this.acceptsNotification = acceptsNotification;
    }

    public Boolean getAcceptsNotification() {
        return acceptsNotification;
    }
}

File: als-server/shared/src/main/scala/org/mulesoft/als/server/feature/serialization/SerializationClientCapabilities.scala
package org.mulesoft.als.server.feature.serialization

/** Capabilities specific to `workspace/serializedModel`.
  *
  * @param acceptsNotification
  *   Whether the clients accepts notification with the serialized resolved model.
  */
case class SerializationClientCapabilities(acceptsNotification: Boolean = false)
